server/loginserver.ts

```diff
@@ -37,6 +37,7 @@ function parseJSON(json: string) {
 }
 
 type LoginServerResponse = [AnyObject | null, number, Error | null];
+type response = NodeJS.ReadableStream | {statusCode: string};
```

Comment:
We typically use Capitalized types. Also, this should be `&`, not `|`.

Reply:
That code doesn't exist anymore. It's from an old commit.

Reply:
It's now an interface